src/main.rs

```diff
@@ -221,6 +223,15 @@ fn run_init(cfg: &Config) -> ! {
 fn main() {
     let cfg = make_config_from_cli();
 
+    let root_dir = open(
+        &cfg.rootfs,
+        OFlag::O_RDONLY | OFlag::O_DIRECTORY | OFlag::O_CLOEXEC,
+        Mode::from_bits(0o777).expect("what?"),
```

Comment:
`.expect("what?")` can be replaced by `.unwrap()` ;D

Reply:
oh I didn't remove that